DependencyInjection/SonataMediaExtension.php

```diff
@@ -435,6 +439,42 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
     }
 
     /**
+     * Set Category && Category Manager In Media Bundle.
+     *
+     * @param ContainerBuilder $container
+     * @param array            $config
+     * @param XmlFileLoader    $loader
+     */
+    public function configureCategoryInMedia(ContainerBuilder $container, array $config, XmlFileLoader $loader)
+    {
+        // Use default sonata-project classification if present and nothing against
+        if (
+            false === $config['force_disable_category']
+            && interface_exists('Sonata\ClassificationBundle\Model\CategoryInterface')
```

Comment:
Nope, this is relying on "if Class exists" so if you have the class in composer bootstrap this will be true.